samples/dotnet/EventHub/KafkaTriggerWithHeaders.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Kafka;
+using Microsoft.Azure.WebJobs.Extensions.Storage;
+using Microsoft.Extensions.Logging;
+
+namespace Eventhub
+{
+    public class KafkaTriggerSingleWithHeaders
+    {
+        [FunctionName("KafkaTriggerSingleWithHeaders")]
+        public static void Run(
+            [KafkaTrigger("BrokerList",
```

Comment:
Shouldn't BrokerList also be mentioned as %BrokerList% here?

Reply:
BrokerList and %BrokerList% both work.